app/assets/stylesheets/content/_tables.sass

```diff
@@ -60,14 +106,96 @@ tr
     td
       &.category
         white-space: normal
+      &.subject
+        text-align: left
+    &.idnt td.subject .icon-context:before
+      padding: 9px 0 0 10px
 
-tr.issue
-  td
-    &.subject
+  &.work-package
+    white-space: nowrap
+    &.sum
+      font-weight: bold
+    td.subject
+      white-space: normal
       text-align: left
-  &.idnt td.subject a
-    background: image-url("bullet_arrow_right.png") no-repeat 0 50%
-    padding-left: 16px
+    td.done_ratio table.progress
+      margin-left: auto
+      margin-right: auto
+    &.idnt td.subject.icon-context
+      display: inherit
+    &.issue td.category
+      white-space: normal
+
+  &.entry
+    border: 1px solid #f8f8f8
+    td
+      white-space: nowrap
+      &.filename
+        width: 30%
+      &.size
+        text-align: right
+        font-size: 90%
+      &.revision, &.author
+        text-align: center
+      &.age
+        text-align: right
+
+  div.expander
+    cursor: pointer
+    float: left
+
+  &.open span .expand, &.closed span .collapse
+    display: none
+
+  &.changeset td
+    &.author, &.committed_on
+      text-align: center
+      width: 15%
+
+  [class*=reorder]
+    .icon-context:before
+      padding: 0 0 0 0
+    .up
+      transform: rotate(90deg)
+    .down
+      transform: rotate(270deg)
```

Comment:
Does not work as I would have expected

![image](https://cloud.githubusercontent.com/assets/453584/9513981/35d06a2a-4c96-11e5-8316-32258aac1fe0.png)